app/views/staff/shared/xml/_forecast.xml.haml

```diff
@@ -0,0 +1,7 @@
+%planned-disbursement{"type" => forecast.forecast_type}
+  %period-start{"iso-date" => forecast.period_start_date}
+  - if forecast.period_end_date.present?
+    %period-end{"iso-date" => forecast.period_end_date}
+  %value{"currency" => forecast.currency, "value-date" => forecast.period_start_date}= forecast.value
+  %provider-org{"provider-activity-id" => "", "type" => forecast.providing_organisation_type, "ref" => forecast.providing_organisation_reference}
+    %narrative=forecast.providing_organisation_name
```

Comment:
I have no massive opinions either way, but I wonder if this partial _should_ be called `planned_disbursement`, as that's what we're returning in the XML

Reply:
Possibly? :scratches head: